### ISSUE TITLE
InputMask: Unmask prop causes issues

### ISSUE THREAD

**Issue by vanpyrzericj** [primefaces/primevue#6261]
### Describe the bug

When adding the `unmask` prop to an `InputMask` component, instead of just "unmasking" the data model's value (v-model) while keeping the visual display with the mask, instead the mask disappears for the user when entering their input, and only after clicking out of focus of the element does the mask reappear. This defeats the purpose of the input mask.

### Reproducer

https://stackblitz.com/edit/jvufqw?file=src%2FApp.vue

### PrimeVue version

4.0.4

### Vue version

4.x

### Language

TypeScript

### Build / Runtime

Vite

### Steps to reproduce the behavior

Create an `InputMask` component and set the `unmask` prop to true. Enter text into the input field.

### Expected behavior

The end-user should visually see no difference between what `unmask=true` and `unmask=false` is set to.

**Comment by tugcekucukoglu** [comment]
Please check the latest version. If you think the problem still persists, please create a new issue with a reproducer.
